src/rebar_prv_dialyzer.erl

```diff
@@ -353,8 +353,19 @@ update_base_plt(State, BasePlt, Output, BaseFiles) ->
             build_plt(State, BasePlt, Output, BaseFiles)
     end.
 
+build_plt(State, Plt, _, []) ->
+    ?INFO("Building with no files in ~p...", [Plt]),
+    Opts = [{get_warnings, false},
+            {output_plt, Plt},
+            {apps, [erts]}],
+    % Create a PLT with erts files and then remove erts files to be left with an
+    % empty PLT. Dialyzer will crash when trying to build a PLT with an empty
+    % file list.
+    _ = dialyzer:run([{analysis_type, plt_build} | Opts]),
```

Comment:
In reply to https://github.com/erlang/rebar3/pull/1262#discussion_r70266722

That's exactly right, I have added a comment